master/reference/cni-plugin/configuration.md

```diff
@@ -100,8 +100,8 @@ By default, {{site.prodname}} IPAM will assign IP addresses from all the availab
 
 Optionally, the list of possible IPv4 and IPv6 pools can also be specified via the following properties:
 
-* `ipv4_pools`: An array of CIDR strings (e.g. `"ipv4_pools": ["10.0.0.0/24", "20.0.0.0/16"]`)
-* `ipv6_pools`: An array of CIDR strings (e.g. `"ipv6_pools": ["2001:db8::1/120"]`)
+* `ipv4_pools`: An array of CIDR strings, or pool names. (e.g. `"ipv4_pools": ["10.0.0.0/24", "20.0.0.0/16", "default-ipv4-ippool"]`)
```

Comment:
Here we have extra commas and missing commas: should be no comma after `strings` yet there should be a comma after `e.g.` (lines 103 and 104)